src/components/vedlegg/EttersendelseView.tsx

```diff
@@ -174,7 +174,7 @@ const EttersendelseView: React.FC<Props> = ({restStatus}) => {
             setIsUploading(false);
         };
         const handleFileUploadFailed = () => {
-            dispatch(hentInnsynsdata(BACKEND_FEIL_ID, InnsynsdataSti.VEDLEGG, false));
+            dispatch(hentInnsynsdata(fiksDigisosId ?? "", InnsynsdataSti.VEDLEGG, false));
```

Comment:
Hva skjer når vi sender dispatch med tom string?

Reply:
Ka du trur @joeldlin ?

Reply:
dispatchen fetcher fra `/innsyn/${fiksDigisosId}/${sti}` så hvis fiksDigisosId er "" vil jo ikke urlen bli gyldig..(
 /innsyn//vedlegg )

Reply:
Jeg er usikker på om det blir å gå greit om `?? ""` fjernes, ser fra tidligere av at `hentInnsynsdata` var satt opp sånn her : `hentInnsynsdata(fiksDigisosId: string | string....` om det har noe å si.

Ser også at `fiksDigisosId` er satt til `undefined` i `initialState` i `innsynsDataReducer.ts`. Får anta at det er grunnen for `?? ""`.

Mene kan jo prøve å fjerne det og se hva som skjer?